### DISCUSSION TITLE
Lab 2- Extra space in "Body" of html

### DISCUSSION THREAD

**Discussion by ghost** [albertkun/21S-ASIAAM-191A#71] [Help Desk] [answered]
Hi all.
I am working on finishing up this lab assignment but after adding in my image and ordered list this odd extra space appeared in between 2 div classes in the body of my `index.html`. I am not sure if this is a problem with my code or if I need to add something in order for this area to be filled but I was hoping for some suggestions if anybody has dealt with this! I added a photo of the space I am referring to and can add screenshots of my `index.html` too if that will help.

<img width="1434" alt="Screen Shot 2021-04-11 at 10 47 20 PM" src="https://user-images.githubusercontent.com/81720023/114342306-dcbfa680-9b18-11eb-9278-51dbdc4d59f1.png">

**Comment by albertkun** [comment, marked as answer]
@ktgreenler this is probably because of the `css grid` that is being used in the template and how your extra html is falling outside of it.  I didn't cover `css grid` in detail yet, so if you want to keep things simple you can takeout `style.css` code. Without looking at your code, it is hard to tell, but if you paste your index.html we can take a look. If you do paste your code be sure to use the three ``` symbols at the top and bottom, like so:

_```html_

put your html in here!

_```_



Also, if you want to get ahead start with css-grid here's a tutorial, but we will cover this in week 5.

[https://css-tricks.com/getting-started-css-grid/](https://css-tricks.com/getting-started-css-grid/)

**Reply by ghost** [reply]
Hi @albertkun. After looking at the link you provided I was able to find a quick fix! If anyone else has this problem I suggest reading through the above link and maybe looking at the `grid_template_rows` line in your `style.css`. I look forward to understanding this better when we cover it in class. Thank you!

**Reply by albertkun** [reply]
@ktgreenler That's great to hear that you found a solution to it using the referenced link! And nice job using the ` marks for the inline code! 😄
